docs/source/conf.py

```diff
@@ -101,6 +101,7 @@
     "check_error",
     "cudart",
     "is_bf16_supported",
+    "is_tf32_supported",
```

Comment:
Adding new entries in this file is not ok. If it is a public API you need to make sure it is properly documented on the website. If it should be private, you can make it so by defining it with a leading _

Reply:
@albanD Got it! I was imitating what was done for `is_bf16_supported` mechanically; I'll document!

Reply:
Thanks!